include/nbl/builtin/glsl/ext/FFT/parameters.glsl

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_GLSL_EXT_FFT_PARAMETERS_INCLUDED_
+#define _NBL_GLSL_EXT_FFT_PARAMETERS_INCLUDED_
+
+struct nbl_glsl_ext_FFT_Parameters_t
+{
+    uvec4   dimension; // settings packed into the w component
+    uvec3   padded_dimension;
```

Comment:
btw even under `std140` rules if you define it as the following
```glsl
uvec3 dimension;
uint flags;
uvec3 padded_dimension;
```
the data layout won't change

Reply:
I tested that first and didn't work, I completely expected it to work. 
When I used renderdoc the data was just weirdly wrong

Reply:
might have been caused by https://github.com/Devsh-Graphics-Programming/Nabla/pull/63#discussion_r576138293

or an engine bug @Crisspl ?

Reply:
this struct is used in some buffer or push constants or? anyway i dont see a chance for engine bug here

Reply:
@devshgraphicsprogramming  @Crisspl 
I remember I tried many things including your suggestion to add padding to the end. 

I was confused a lot because looking at my renderdoc capture I saw many correct values for the push constants for the first dispatches but for the last dispatches the uint in the middle had wrong values. 
maybe because I used the same pipelinelayout(?) I have no idea.

I just tested this again and It didn't work even with the padding at the end

Reply:
hmmm.. maybe engine bug then, would need to inspect further. But it would need some working code presenting  this behaviour